### PR TITLE
Add google-analytics job

### PR DESCRIPTION
**Pull request by maxiberta**
Credential variables `GA_VIEW_ID` and `GA_KEY_FILE_LOCATION` should be setup as described in the [Google Analytics Reporting API v4 quickstart](https://developers.google.com/analytics/devguides/reporting/core/v4/quickstart/service-py).

### CODE REVIEW

File: metrics/metrics.yaml [hunk 1]
```diff
@@ -131,3 +131,26 @@
...
+      - shell: |
+          #!/bin/bash
+          set -e
+          rm -rf *
+          git clone https://github.com/canonical-server/metrics.git
+          cd metrics
+          python -m metrics.google_analytics
+    parameters:
+      - string:
+          name: HTTPS_PROXY
```
Comment (powersj) [inline metrics/metrics.yaml]: Do you actually need this set?

File: metrics/metrics.yaml [hunk 2]
```diff
@@ -131,3 +131,26 @@
...
+          default: https://squid.internal:3128
+    wrappers:
+     - credentials-binding:
```
Comment (powersj) [inline metrics/metrics.yaml]: What else is required to get this setup?

**Comment by powersj** [comment]
@maxiberta my apologies for forgetting about this merge request. Couple points:

* There are some conflicts now, which should be easy to resolve.
* You should also take a look at the metric file and see how we have a generic builder that you can use to simplify the job.
* Couple in-line comments

File: metrics/metrics.yaml (hunk 1 above)
Comment (maxiberta) [inline metrics/metrics.yaml]: I think so. You need this to be able to hit Google servers, if running from the internal network.

Comment (maxiberta) [inline metrics/metrics.yaml]: Oh, btw, proxy should allow connections to: analytics.googleapis.com, analyticsreporting.googleapis.com, accounts.google.com.

File: metrics/metrics.yaml (hunk 2 above)
Comment (maxiberta) [inline metrics/metrics.yaml]: I believe these 2 variables should be defined as described in the [credentials binding plugin page](https://wiki.jenkins.io/display/JENKINS/Credentials+Binding+Plugin) (examples [here](https://support.cloudbees.com/hc/en-us/articles/203802500-Injecting-Secrets-into-Jenkins-Build-Jobs)).

**Comment by maxiberta** [comment]
@powersj Conflicts resolved. Sorry, what metrics file? Thanks!

File: metrics/metrics.yaml (hunk 1 above)
Comment (powersj) [inline metrics/metrics.yaml]: Ok! I was able to connect over port 443 to all three domains.

File: metrics/metrics.yaml (hunk 2 above)
Comment (powersj) [inline metrics/metrics.yaml]: ok - let's give this a shot :)

**Pull request merged**